src/Serilog.Sinks.MSSqlServer/Sinks/MSSqlServer/MSSqlServerSink.cs

```diff
@@ -303,6 +303,17 @@ private string ConvertPropertiesToXmlStructure(IEnumerable<KeyValuePair<string,
             if (options.ExcludeAdditionalProperties)
                 properties = properties.Where(p => !_additionalDataColumnNames.Contains(p.Key));
 
+            if (options.PropertiesFilter != null)
+                try
```

Comment:
Not strictly required, but braces around the body of this `if` block would make the intention a little clearer.